mod_conf_sql.c

```diff
@@ -816,13 +832,18 @@ static int sqlconf_read_db(pool *p) {
   destroy_pool(cmd->pool);
 
   /* Define the connection we'll be making. */
-  cmd = sqlconf_cmd_alloc(p, 4, "sqlconf", sqlconf_db.user, sqlconf_db.pass,
-     pstrcat(p, sqlconf_db.database, "@", sqlconf_db.server, NULL));
+  if (!strncasecmp(sqlconf_db.database,"sqlite",6)==0)
+  {
```

Comment:
The use of an open curly brace on its own line is not in keeping with the coding style that ProFTPD uses.  It's best if all of the code uses the same consistent style, as it makes sight-reading the code much easier.